lib/resque/failure/multiple.rb

```diff
@@ -25,7 +25,7 @@ def save
         exception = nil
         @backends.each do |b|
           begin
-            b.save
+            b.save unless b.class&.get_blacklisted_exceptions.include?(exception.class.to_s)
```

Comment:
I'd suggest to add a new method for this, probably named `is_blacklisted_exception.`

Reply:
Right, i will add the new method to the base class since this is something all backends have in common!